xdsl/syntax_printer.py

```diff
@@ -9,10 +9,12 @@ class SyntaxPrinter(Printer, ColorPrinter):
     A printer for printing syntax-highlighted mlir code to a terminal.
     """
 
```

Comment:
Any reason for removing the colouring of SSA values? Are you following some particular colour scheme? For reference I was roughly following https://github.com/llvm/llvm-project/blob/8b3a124ad87d1e808852644090ea5d1117fe2f9f/mlir/utils/emacs/mlir-mode.el#L36